src/api/CheckListItemApi.ts

```diff
@@ -8,7 +8,7 @@ import {
   useQueryClient,
 } from "@tanstack/react-query";
 import { IPerson, Person, useCurrentUser } from "api/UserApi";
-import { RoleType } from "./RolesApi";
+import { RoleType } from "api/RolesApi";
```

Comment:
Not related to this line, but to the file... do we maybe want to consider grouping our APIs into folders.  So the different CRUD hooks are under the respective folders?